### PR TITLE
Clear CVEs

### PR DESCRIPTION
**Pull request by ryw**
Running `npm update` doesn't reduced the CVE list :(

* NODE-SECURITY-723 | critical | 9.00 | pem        | 1.4.4    | fixed in >=1.13.2      
* NODE-SECURITY-663 | critical | 9.00 | open       | 0.0.5    | fixed in >0.0.5   
* CVE-2019-10747    | high     | 9.80 | set-value  | 2.0.0    | fixed in >=2.0.1 <3.0.0 || >=3.0.1   
* CVE-2019-10746    | high     | 9.80 | mixin-deep | 1.2.0    | fixed in >=1.3.2 <2.0.0 || >=2.0.1   
* CVE-2019-10744    | high     | 9.80 | lodash     | 4.16.4   | fixed in >=4.17.12       
* CVE-2019-10744    | high     | 9.80 | lodash     | 3.10.1   | fixed in >=4.17.12    
* CVE-2019-10744    | high     | 9.80 | lodash     | 3.9.3    | fixed in >=4.17.12       
* CVE-2019-10744    | high     | 9.80 | lodash     | 4.17.4   | fixed in >=4.17.12        
* CVE-2019-10744    | high     | 9.80 | lodash     | 4.17.11  | fixed in >=4.17.12   
* CVE-2019-10744    | high     | 9.80 | lodash     | 4.17.5   | fixed in >=4.17.12    
* CVE-2019-10744    | high     | 9.80 | lodash     | 4.0.1    | fixed in >=4.17.12    
* CVE-2018-16487    | high     | 9.80 | lodash     | 4.17.5   | fixed in >=4.17.11  
* CVE-2018-16487    | high     | 9.80 | lodash     | 4.17.4   | fixed in >=4.17.11   
* CVE-2018-16487    | high     | 9.80 | lodash     | 4.0.1    | fixed in >=4.17.11             
* CVE-2018-16487    | high     | 9.80 | lodash     | 4.16.4   | fixed in >=4.17.11      
* CVE-2018-16487    | high     | 9.80 | lodash     | 3.10.1   | fixed in >=4.17.11    
* CVE-2018-16487    | high     | 9.80 | lodash     | 3.9.3    | fixed in >=4.17.11        
* CVE-2019-17592    | high     | 7.50 | csv-parse  | 2.5.0    | fixed in >=4.4.6        
* CVE-2019-17592    | high     | 7.50 | csv-parse  | 1.2.4    | fixed in >=4.4.6         
* CVE-2018-3737     | high     | 7.50 | sshpk      | 1.7.0    | fixed in >=1.13.2 < 1.14.0 || >=1.14.1 
* CVE-2017-16023    | high     | 7.50 | decamelize | 1.1.1    | fixed in >=1.1.2

### CODE REVIEW

**Comment by andriisoldatenko** [comment]
@ryw can you please retest?

**Comment by andriisoldatenko** [comment]
@ash @schnie i decided to refactor using vanilla nginx config, wdyt?
cc @sjmiller609

**Comment by andriisoldatenko** [comment]
practical tests:
```
 curl -L -X GET -v -H "x-code: 502" -H "x-ingress-name: airflow-ingress" http://localhost:8080/tesd
Note: Unnecessary use of -X or --request, GET is already inferred.
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 8080 (#0)
> GET /tesd HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.54.0
> Accept: */*
> x-code: 502
> x-ingress-name: airflow-ingress
>
< HTTP/1.1 502 Bad Gateway
< Server: nginx/1.17.10
< Date: Tue, 26 May 2020 11:46:15 GMT
< Content-Type: text/html
< Content-Length: 1048
< Connection: keep-alive
< ETag: "5eab43fd-418"
<
<!DOCTYPE html>
<html>
  <head>
    <title>Astronomer - 502</title>
    <link rel="icon" type="image/x-icon" href="https://d34umsbsd4n9ln.cloudfront.net/static/astro.ico">
    <link href='https://fonts.googleapis.com/css?family=Work+Sans' rel='stylesheet' type='text/css'>
  </head>
  <body style="background-color: #fefcf9; font-family: 'Work Sans', sans-serif;">
    <section style="padding-top: 3rem; text-align: center; box-sizing: border-box; color: #069dac;">
      <div style="width:100%; margin: auto; max-width: 56rem; padding: 0 1.5rem; box-sizing: border-box;">
        <img src="https://d34umsbsd4n9ln.cloudfront.net/static/astronaut.svg" style="height: 30em">
        <h1 style="font-size: 2.5rem; line-height: 1.13em; margin: 2rem 0 1rem; box-sizing: border-box;">Airflow webserver is down</h1>
        <p style="font-size: 1.2rem;">Check back in a few seconds</p>
        <a href="javascript:history.back()" style="color: #f4362c; text-decoration: none; ">&lt;<!-- --> Back to Earth</a>
      </div>
    </section>
  </body>
</html>
* Connection #0 to host localhost left intact
```

**Comment by andriisoldatenko** [comment]
```
curl -L -X GET -v -H "x-ingress-name: airflow-ingress" http://localhost:8080/healthz
Note: Unnecessary use of -X or --request, GET is already inferred.
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 8080 (#0)
> GET /healthz HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.54.0
> Accept: */*
> x-ingress-name: airflow-ingress
>
< HTTP/1.1 200 OK
< Server: nginx/1.17.10
< Date: Tue, 26 May 2020 11:51:13 GMT
< Content-Type: text/plain
< Content-Length: 0
< Connection: keep-alive
<
* Connection #0 to host localhost left intact
```

**Comment by ashb** [comment]
Yeah, using node for serving static content seems like overkill

**Comment by andriisoldatenko** [comment]
./twistcli scan:
```
Vulnerabilities found for image test:latest: total - 28, critical - 0, high - 0, medium - 16, low - 12
Vulnerability threshold check results: PASS
Compliance found for image test:latest: total - 0, critical - 0, high - 0, medium - 0, low - 0
Compliance threshold check results: PASS
```

File: include/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        location /503.html {
+            root /etc/nginx/html/airflow;
+            allow all;
+        }
+
+        location /healthz {
+            return 200;
+        }
+    }
+}
```
Comment (andriisoldatenko) [inline include/nginx.conf:55]:
```suggestion
}

```

**Comment by ryw** [comment]
This PR eliminates all CVEs (thanks @andriisoldatenko!)

@schnie can you review

**Comment by ryw** [comment]
i'll fix snyk once this is merged

**Comment by schnie** [comment]
If we're removing node should we delete the whole `src` directory too?

**Comment by schnie** [comment]
Well, maybe not the whole directory, but delete the js code and maybe move the html files up to a top-level `static` directory or something. Just so the code structure reflects how its running now.

**Comment by andriisoldatenko** [comment]
@schnie PTAL.

**Comment by andriisoldatenko** [comment]
@schnie done. (practically also works), just rechecked.

**Pull request merged**